### ISSUE TITLE
Bundle plotutils?

### ISSUE THREAD

**Issue by beckermr** [biggles-plot/biggles#81]
i am not 100% sure about this one.

**Comment by esheldon** [comment]
It would be a big win ergonomically.  Every time I do a new install I need to build plotutils as well, which is a pain.

**Comment by beckermr** [comment]
Yeah. I just don't want to be responsible for compiling and linking against X11. This is already a pain on a mac. I submitted a patch to the homebrew bits that do this for this exact reason.
